scenario/fluid_block/fluid_block.py

```diff
@@ -0,0 +1,207 @@
+"""Fluid Block scenario."""
+from functools import singledispatchmethod
+import os
+from typing import List, Optional
+import shutil
+
+from inductiva import tasks, resources, scenarios, simulators
```

Comment:
I prefer import inductiva only.
@sergiomsantos : what is your opinion?

Reply:
me too; better namespace handling.